### PR TITLE
ref(analytics): convert user feedback

### PR DESCRIPTION
**Pull request by scefali**

### CODE REVIEW

File: static/app/views/userFeedback/userFeedbackEmpty.tsx [hunk 1]
```diff
@@ -44,23 +44,22 @@ export function UserFeedbackEmpty({projectIds}: Props) {
 
     if (hasAnyFeedback === false) {
       // send to reload only due to higher event volume
-      trackAdhocEvent({
-        eventKey: 'user_feedback.viewed',
-        org_id: parseInt(organization.id, 10),
-        projects: projectIds,
```
Comment (scefali) [inline static/app/views/userFeedback/userFeedbackEmpty.tsx:50]: Note we really need to convert to a string here

File: static/app/views/userFeedback/userFeedbackEmpty.tsx [hunk 2]
```diff
@@ -44,23 +44,22 @@ export function UserFeedbackEmpty({projectIds}: Props) {
 
     if (hasAnyFeedback === false) {
       // send to reload only due to higher event volume
```
Comment (malwilley) [inline static/app/views/userFeedback/userFeedbackEmpty.tsx:46]: Is this comment still relavent?

Comment (scefali) [reply static/app/views/userFeedback/userFeedbackEmpty.tsx:46]: @malwilley yes

Comment (malwilley) [reply static/app/views/userFeedback/userFeedbackEmpty.tsx:46]: Ah okay I see you did make the event name null above, cool

**Review by malwilley — APPROVED** [review]

**Pull request merged**